docker/build-tool/Dockerfile

```
@@ -0,0 +1,54 @@
+FROM debian:buster
+COPY ./dev_setup.sh /build_setup.sh
+COPY ./rust-toolchain.toml /rust-toolchain.toml
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update -y && \
+    apt-get install -yq \
+        build-essential \
+        curl \
+        file \
+        git \
+        libssl-dev \
+        pkgconf \
+        sudo \
+        unzip \
+        xutils-dev \
+        protobuf-compiler \
+        lcov \
+        python3 \
+        coreutils \
+        default-mysql-client \
+        clang \
+        llvm \
+        python3-all-dev \
+        python3-setuptools \
+        python3-pip \
+        && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* && \
+    useradd rust --user-group --create-home --shell /bin/bash --groups sudo
+# We use the instructions at https://github.com/rust-lang/rustup/issues/2383
+# to install the rustup toolchain as root.
+ENV RUSTUP_HOME=/opt/rust/rustup \
+    PATH=/home/rust/.cargo/bin:/opt/rust/cargo/bin:/usr/local/musl/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+RUN curl https://sh.rustup.rs -sSf -sSf |\
+    env CARGO_HOME=/opt/rust/cargo \
+        sh -s -- -y --default-toolchain stable --profile minimal --no-modify-path && \
+    export RUST_VERSION="$(awk -F'[ ="]+' '$1 == "channel" { print $2 }' /rust-toolchain.toml)" && \
+    env CARGO_HOME=/opt/rust/cargo rustup install $RUST_VERSION && \
+    env CARGO_HOME=/opt/rust/cargo rustup set profile minimal && \
+    env CARGO_HOME=/opt/rust/cargo rustup component add rustfmt --toolchain $RUST_VERSION && \
+    env CARGO_HOME=/opt/rust/cargo  rustup component add rust-src --toolchain $RUST_VERSION && \
```

```suggestion
    env CARGO_HOME=/opt/rust/cargo rustup component add rust-src --toolchain $RUST_VERSION && \
```

How about keeping the same indent?

---

looking for a Dockerfile linter on clion...